docs/coordinates/performance.inc.rst

```diff
@@ -56,3 +56,157 @@ properties::
 
 ..
   EXAMPLE END
+
+
+Improving Performance for Arrays of ``obstime``
+-----------------------------------------------
+
+The most expensive operations when transforming between observer-dependent coordinate
+frames (e.g. ``AltAz``) and sky-fixed frames (e.g. ``ICRS``) are the calculation
+of the orientation and position of Earth.
+
+If |skycoord| instances with a large ``obstime`` array are transformed,
+these calculations can be sped up by factors up to 100, whilst still keeping micro-arcsecond precision,
+by utilizing interpolation instead of calculating Earth orientation parameters for each individual point.
+
+This can be achieved through the ``erfa_astrom`` state and the ``ErfaAstromInterpolator``
+class like this
+
+..
+  EXAMPLE START
+  Improving performance for obstime arrays
+
+To use interpolation for the astrometric values in coordinate transformation, use::
+
+   >>> from astropy.coordinates import SkyCoord, EarthLocation, AltAz
+   >>> from astropy.coordinates.erfa_astrom import erfa_astrom, ErfaAstromInterpolator
+   >>> from astropy.time import Time
+   >>> from time import perf_counter
+   >>> import numpy as np
+   >>> import astropy.units as u
+
+
+   >>> # array with 10000 obstimes
+   >>> obstime = Time('2020-01-01T20:00') + np.linspace(0, 6, 10000) * u.hour
```

Comment:
I'd pick an earlier time so remote data is not necessary.

Reply:
I used a 2010 time and it still required remote data